### PR TITLE
backend tests: inventory API tests

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issues: https://tracker.mender.io/browse/MEN-2403

### CODE REVIEW

File: backend-tests/tests/test_inventory.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        # filter devices
+        qs_params={}
+        qs_params['per_page'] = 100
+        qs_params['mac'] = 'regex:06:2'
+        r = invm.with_auth(utoken).call('GET',
+                                        inventory.URL_DEVICES,
+                                        qs_params=qs_params)
+        assert r.status_code == 200
+        api_devs = r.json()
+        assert len(api_devs) == 11
```
Comment (lluiscampos) [inline backend-tests/tests/test_inventory.py]: Why 11? Shouldn't be 10? Is it matching :06:02?

Comment (lluiscampos) [reply backend-tests/tests/test_inventory.py]: Ok, I see it now: it matches a :06:2 mac address. As they represent mac addresses, I was expecting the last octet of the mac to be formatted with 0s, like 01, 02...

File: backend-tests/tests/common.py [hunk 2]
```diff
@@ -83,13 +84,28 @@ def create_tenant(name):
...
+        page = page + 1
+        qs_params['page'] = page
+        qs_params['per_page'] = per_page
+        r = api.call('GET', tenantadm.URL_INTERNAL_TENANTS, qs_params=qs_params)
+        assert r.status_code == 200
+        api_tenants = r.json()
+
+        found = [at for at in api_tenants if at['id'] == id]
+        if len(found) > 0:
+            found = found
```
[the remark this answers is not in the document]
Comment (kjaskiewiczz) [reply backend-tests/tests/common.py]: fixed

[the remark this answers is not in the document]
Comment (lluiscampos) [reply backend-tests/tests/common.py]: Thanks

**Pull request merged**